packages/odyssey-react/src/components/Tab/index.tsx

```diff
@@ -166,7 +211,7 @@ Tabs.Tab = function TabsTab ({ children, id, ariaControls, selected, onClick })
   )
 }
 
-Tabs.PanelContainer = ({ children }) => (<div className="ods-tabs--tabpanel">{children}</div>)
+Tabs.PanelContainer = ({ children }) => (<div className="ods-tabs--tabpanel" data-testid="ods-panel-container">{children}</div>)
```

Comment:
Do we want this attr in the actual component?

Reply:
We don't. Thanks for catching, it has been removed.